docs/team/teamler.json

```diff
@@ -3197,12 +3197,10 @@
     "name": "Leg0shi_",
     "uuid": "08521f46-8bb8-43a8-9eb3-d3842c34473d",
     "sex": "male",
-    "fields": [
-      "Plugin Development"
-    ],
     "rank_history": {
       "initial": "youtuber",
       "03.06.2023": "developer"
```

Comment:
Hier fehlt ein , ,ansonsten wird es einen Fehler geben.

Reply:
wo genau, es ist ja ein , in 3199 und die nächste zeile ist 3202?